linker/shared/src/main/scala/org/scalajs/linker/backend/wasmemitter/CoreWasmLib.scala

```diff
@@ -309,7 +309,7 @@ final class CoreWasmLib(coreSpec: CoreSpec) {
       )
     }
 
-    addGlobalHelperImport(genGlobalID.jsLinkingInfo, RefType.any)
+    addGlobalHelperImport(genGlobalID.fileLevelThis, RefType.any)
```

Comment:
```suggestion
    addGlobalHelperImport(genGlobalID.fileLevelThis, RefType.anyref)
```
In theory, the global `this` value can be any value, including `null`.